client/flutter/lib/components/page_scaffold/page_header.dart

```diff
@@ -23,33 +23,40 @@ class PageHeader extends StatelessWidget {
       leading: Container(),
       expandedHeight: 120,
       backgroundColor: Colors.white,
-      flexibleSpace: FlexibleSpaceBar(background:_buildHeader()),
+      flexibleSpace: FlexibleSpaceBar(background: _buildHeader()),
     );
   }
 
   Widget _buildHeader() {
     List<Widget> headerItems = [
-      if (this.showBackButton) BackArrow(),
-      
-      Column(
+      if (this.showBackButton)
+        Transform.translate(offset: Offset(-12, 0), child: BackArrow()),
+      Expanded(child: Column(
         mainAxisAlignment: MainAxisAlignment.center,
-        crossAxisAlignment: CrossAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.stretch,
         children: <Widget>[
+          //Todo: Decide what to do when title text overflow
           Text(this.title,
-              textScaleFactor: 1.8,
+              maxLines: 1,
+              overflow: TextOverflow.fade,
+              softWrap: false,
               style: TextStyle(
-                  color: Colors.blueAccent, fontWeight: FontWeight.bold)),
+                  color: Color(0xff1A458E),
```

Comment:
probably out of scope of this PR (and out of scope of v1.0 since we are no longer refactoring), but we will want to pull this out into a reusable constant

Reply:
Yeah agree, I proposed that in the #eng slack. I think it will improve a lot the consistency of the app. After v1.0 I would be happy to submit a pull request grouping all the text styles in variables or following the Flutter guidelines in TextThemes